articles/quickstarts/index.md

```diff
@@ -19,17 +19,150 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
+   - Or, if you prefer to use PowerShell or pwsh: open a shell, run `conda init powershell`, then close and re-open the shell.
+1. Create and activate a new conda environment named `qsharp-env` with the required packages (including Jupyter Notebook and IQ#) by running the following commands:
+    ```
+    conda create -n qsharp-env -c quantum-engineering qsharp notebook
+    conda activate qsharp-env
+    ```
+1. Run `python -c "import qsharp"` from the same terminal to verify your installation and populate your local package cache with all required QDK components.
+
+To start writing Q# code with Python, see [Develop with Q# and Python](xref:microsoft.quantum.install.python).
 
-To proceed with installing the QDK and creating Q# projects, select your preferred setup:
+### [Install using .NET CLI (advanced)](#tab/tabid-dotnetcli)
 
-[Develop with Q# applications](xref:microsoft.quantum.install.standalone) - Choose this approach to work with Q# from the command prompt. This does not require a driver or a host program like the below options.
+1. Prerequisites:
 
-[Develop with Q# Jupyter Notebooks](xref:microsoft.quantum.install.jupyter) - Select this environment to run Q# code in cells with embedded text or create quantum computing interactive tutorials. 
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - [Jupyter Notebook](https://jupyter.readthedocs.io/en/latest/install.html)
+    - [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
 
-[Develop with Q# and Python](xref:microsoft.quantum.install.python) - Enables you to combine Python and Q# to create a Python host program that calls Q# operations.
+1. Install the `Microsoft.Quantum.IQSharp` package.
 
-[Develop with Q# and .NET](xref:microsoft.quantum.install.cs) - Combine C#, F#, or VB.NET with Q# to create a .NET host program that calls Q# operations.
+    ```dotnetcli
+        dotnet tool install -g Microsoft.Quantum.IQSharp
+        dotnet iqsharp install
+    ```
+
+To start writing Q# code with Jupyter Notebooks, see [Develop with Q# Jupyter Notebooks](xref:microsoft.quantum.install.jupyter).
+
+    > [!NOTE]
+    > - If you get an error during the `dotnet iqsharp install` step, open a new terminal window and try again.
+    > - If this still doesn't work, locate the installed `dotnet-iqsharp` tool (on Windows, `dotnet-iqsharp.exe`) and run: <br>`<path to dotnet-iqsharp> install --user --path-to-tool=<path to dotnet-iqsharp>`,<br> where `<path to dotnet-iqsharp>` should be replaced by the absolute path to the `dotnet-iqsharp` tool in your file system.
+     Typically this will be under `.dotnet/tools` in your user profile folder.
+    > - For additional debugging help, see https://natemcmaster.com/blog/2018/02/02/dotnet-global-tool/.
+
+*** 
+
+You now have the IQ# kernel for Jupyter, which provides the core functionality for compiling and running Q# operations from Q# Jupyter Notebooks.
```

Comment:
Please move the sentence: 'To start writing Q# code with Python, see Develop with Q# and Python.' right after this sentence (and remove it from both tabs above)

Reply:
We would need links to both the "Q# Jupyter Notebooks" and "Q# with Python" pages here, since both environments are covered in this section. Or, as @geduardo suggested in another comment, it may be worth keeping the Q# Jupyter Notebooks and Q# with Python sections separate, rather than combining them here.